storage/pruning/pruningStorer.go

```diff
@@ -590,9 +593,10 @@ func (ps *PruningStorer) SearchFirst(key []byte) ([]byte, error) {
 	ps.lock.RLock()
 	defer ps.lock.RUnlock()
 	for _, pd := range ps.activePersisters {
+		ps.stateStatsHandler.IncrPersister(pd.epoch)
```

Comment:
The Has method accesses the storage so maybe we can cover it as well.

Also the Remove accesses the storage

GetBulk as well accesses the cache